### DISCUSSION TITLE
WEB:DIDs Add RSA encryption

### DISCUSSION THREAD

**Discussion by janpmeyer** [eclipse-edc/Connector#2146] [Ideas]
At this time, the WEB:DID extension only allows the use of EC for the public-key JWK. Since we currently only have the option of creating such a key using RSA for technical reasons, the question arises as to whether it makes sense for the EDC to extend the core with this encryption method for WEB:DIDS. 

What do you think about this?

**Comment by kettenbach-it** [comment]
The problem is that the self-description-signer tool only supports RSA and not EC. See here:

https://github.com/deltaDAO/self-description-signer/issues/21

As a result, it is not possible to provide the EDC with a WEB:DID verified by the tool.

**Comment by jimmarino** [comment]
@janpmeyer Can you please be more specific? What parts of the codebase are you referring to and what does "extend the core" mean? Thx.
